scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,608 @@
+{
+ "cells": [
```

Comment:
we should also provide a README for things like stored (pre-trained) model on GCS and any other relevant information / context.

Reply:
It may be better to just move this notebook to the census directory.